server/odc-service/src/main/java/com/oceanbase/odc/service/permission/database/model/ApplyDatabaseParameter.java

```diff
@@ -47,6 +47,10 @@ public class ApplyDatabaseParameter implements Serializable, TaskParameters {
      * Permission types to be applied for, required
      */
     private List<DatabasePermissionType> types;
+    /**
+     * This property is required by the front end to initiate task again
+     */
+    private String expireTimeValue;
```

Comment:
- `expireTimeValue` --> `validDuration`
- Clearly specify in the comment: This field represents the duration of the permission. It is only used for the front end to re-initiate tasks, and the backend does not rely on it.